stm32/Src/examples/example_adc.c

```diff
@@ -143,9 +143,9 @@ int main(void)
     reading_len = sprintf(output, "Voltage: %f\r\n", voltage_reading);

```

Comment:
I think we should be able to use `SysApp_Init` or whatever it's actually defined as to start the `UTIL_Timer` and give us access to `APP_LOG` instead of having to use `sprintf` which get messy.